app/views/shared/sidebar/_my_modules.html.erb

```diff
@@ -1,21 +1,11 @@
 <% if experiment.active_my_modules.present? %>
   <ul>
-    <% experiment.active_my_modules.sort_by{ |m| m.workflow_order }.each do |my_module| %>
-
-      <li class="leaf <%= "active" if currently_active? my_module %>"
-           data-module-id="<%= my_module.id %>">
-        <span class="tree-link second-indent">
-          <% if currently_active? my_module %>
-              <%= my_module.name %>
-          <% else %>
-            <%= link_to my_module.name, module_action_to_link_to(my_module) %>
-          <% end %>
-          <% if is_canvas? %>
-            <a href="" class="canvas-center-on"><span class="fas fa-map-marker-alt"></span></a>
-          <% end %>
+    <% experiment.active_my_modules.each do |my_module| %>
```

Comment:
Also add caching here?

Reply:
I don't think we need to go that deep and cache such small fragments.